akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -1,8 +1,10 @@
 # Cluster Sharding
 
+@@include[cluster.md](../includes/cluster.md) { #links-from-cluster-to-classic }
```

Comment:
Not quite convinced a block with several links to different cluster tools makes sense, wouldn't it be better with only a link to classic sharding from here?

Reply:
I agree, I think one-to-one link would be better. Maybe similar (with similar wording) as the box in the other direction, and then a specific link under that.